Cargo.toml

```diff
@@ -21,6 +21,7 @@ bitflags = "1.2"
 lazy_static = "1"
 log = "0.4"
 termcolor = "1.1"
+nix = "0.17.0"
```

Comment:
Notes on my/fjps deps specify rules:
- I usually omit the patch version.
- If features are specified, the `[dependencies.foo]` syntax is used.
- They are order alphabetically.
- Ordering: `[dependencies]`; `[dependencies.foo]`; `[dependencies.bar]` (optional = true); path; build; dev